tests/test_services/test_terraform_service.py

```diff
@@ -118,6 +118,30 @@ def test_check_matcha_directory_integrity(tmp_path):
     os.chdir("..")
 
 
+def test_verify_kubectl_config_file(terraform_test_config: TerraformConfig):
```

Comment:
I'm a bit confused about this test. Is this doing verification on the user's real `~/.kube` or a 'fake' one?

Reply:
The real one since we are using `~` in original function.

Reply:
Is there a way we can mock? How about this:

* Pass the base path into `verify_kubectl_config_file`, but have a default value of `~`
* In the test, use a temp dir: https://stackoverflow.com/a/26861230